### DISCUSSION TITLE
Trilium Tab managment in web browser

### DISCUSSION THREAD

**Discussion by chaseadam** [TriliumNext/Trilium#3863] [Q&A] [answered]
I am using firefox to access a hosted Trilium and was only able to find a "close tab" keyboard shortcut which indicated it was only available in desktop mode. When using the UI in a browser, is there a keyboard shortcut to close a trilium tab when I am done with it?

It is possible I am mis-managing my tabs. I use search quite a bit, which triggers a new tab. How do people manage their tabs?

**Comment by zadam** [comment, marked as answer]
You can configure the shortcuts for tab management in the Options:

![image](https://user-images.githubusercontent.com/617641/233860317-d041b112-0ee6-4d23-a830-f03026191ea2.png)

They are empty for web browser by default because they would conflict with the native browser tab management shortcuts.
